### PR TITLE
CORCI-1081 ci: Move Unit Testing from EL7 to EL8

### PR DESCRIPTION
**Pull request by brianjmurrell**

### CODE REVIEW

File: ci/provisioning/post_provision_config_nodes_EL_8.sh [hunk 1]
```diff
@@ -6,6 +6,33 @@ LSB_RELEASE=redhat-lsb-core
 EXCLUDE_UPGRADE=dpdk,fuse,mercury,daos,daos-\*
 
 bootstrap_dnf() {
+    # hack in the removal of group repos
+    version="$(lsb_release -sr)"
+    version=${version%.*}
+    if dnf repolist | grep "repo.dc.hpdd.intel.com_repository_centos-${version}-x86_64-group_"; then
+        rm -f /etc/yum.repos.d/repo.dc.hpdd.intel.com_repository_{centos-8.4,daos-stack-centos-8}-x86_64-group_.repo
```
[the remark this answers is not in the document]
Comment (brianjmurrell) [reply ci/provisioning/post_provision_config_nodes_EL_8.sh:13]: It's a single long string.

File: Jenkinsfile [hunk 2]
```diff
@@ -806,7 +806,7 @@ pipeline {
                                 target: 'el8.3',
                                 daos_pkg_version: daosPackagesVersion("centos8", next_version)
                    }
-                } // stage('Test CentOS 7 RPMs')
+                } // stage('Test CentOS 8.3.2011 RPMs') {
```
Comment (phender) [inline Jenkinsfile:809]: Is this a typo?
```suggestion
                } // stage('Test CentOS 8.3.2011 RPMs')
```

File: Jenkinsfile [hunk 3]
```diff
@@ -914,7 +914,7 @@ pipeline {
                                                         name: 'Fault injection leaks',
                                                         id: 'NLT_client')]
                             junit testResults: 'nlt-junit.xml'
-                            archiveArtifacts artifacts: 'nlt_logs/centos7.fault-injection/'
+                            archiveArtifacts artifacts: 'nlt_logs/centos8.fault-injection/'
                         }
                     }
                 } // stage('Fault inection testing')
```
Comment (phender) [inline Jenkinsfile:920]: Should these match?
```suggestion
                } // stage('Fault inection testing on CentOS 8')
```

File: Jenkinsfile [hunk 4]
```diff
@@ -624,7 +624,7 @@ pipeline {
                         }
                     }
                 } // stage('Unit test Bullseye')
```
Comment (phender) [inline Jenkinsfile:626]: Should these match?
```suggestion
                } // stage('Unit test Bullseye on CentOS 8')
```

File: Jenkinsfile [hunk 5]
```diff
@@ -642,7 +642,7 @@ pipeline {
                         always {
                             unitTestPost artifacts: ['unit_test_memcheck_logs.tar.gz',
                                                      'unit_test_memcheck_logs/*.log'],
-                                         valgrind_stash: 'centos7-gcc-unit-memcheck'
+                                         valgrind_stash: 'centos8-gcc-unit-memcheck'
                         }
                     }
                 } // stage('Unit Test with memcheck')
```
Comment (phender) [inline Jenkinsfile:648]: Should these match?
```suggestion
                } // stage('Unit Test with memcheck on CentOS 8')
```

File: Jenkinsfile [hunk 6]
```diff
@@ -1003,28 +1003,28 @@ pipeline {
         } // stage('Test Hardware')
         stage ('Test Report') {
             parallel {
-                stage('Bullseye Report') {
+                stage('Bullseye Report on CentOS 8') {
```
Comment (phender) [inline Jenkinsfile:1006]: I cannot select line 1033 to add the comment there, but should these comments match, e.g.
```
} // stage('Bullseye Report on CentOS 8')
```

**Review by phender — CHANGES_REQUESTED** [review]

**Review by jolivier23 — DISMISSED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (brianjmurrell) [reply Jenkinsfile:809]: Yeah.  It's in a comment, so benign.

File: Jenkinsfile (hunk 3 above)
Comment (brianjmurrell) [reply Jenkinsfile:920]: Probably, but also a comment that can be cleaned up in a subsequent PR (and might be in a PR I have in progress even).

File: Jenkinsfile (hunk 4 above)
Comment (brianjmurrell) [reply Jenkinsfile:626]: Ditto.

File: Jenkinsfile (hunk 6 above)
Comment (brianjmurrell) [reply Jenkinsfile:1006]: Ideally.

Is it worth holding up a 2.0 blocker to fix these comments?  I will respin if you feel like they need fixing in this PR.

Comment (brianjmurrell) [reply Jenkinsfile:1006]: @phender This PR is not as much of a long-pole as I understood it to be.  I will fix the comments and re-push.

File: ci/provisioning/post_provision_config_nodes_EL_8.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (brianjmurrell) [reply ci/provisioning/post_provision_config_nodes_EL_8.sh:13]: Long string.

File: Jenkinsfile (hunk 2 above)
Comment (phender) [reply Jenkinsfile]: Ah, true.  Then these could have been comments with approval.

**Review by phender — APPROVED** [review]

File: src/include/daos_fs.h [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2018-2021 Intel Corporation.
```
Comment (jolivier23) [inline src/include/daos_fs.h:2]: I just did a copyright update patch so if you merge latest master, that should cut down on these.

**Review by jolivier23 — CHANGES_REQUESTED** [review]

**Comment by mchaarawi** [comment]
please close or convert to draft till it's ready

**Pull request closed without merging**